alacritty_terminal/src/term/mod.rs

```diff
@@ -814,11 +814,11 @@ pub struct Term<T> {
     /// arrays. Without it we would have to sanitize cursor.col every time we used it.
     input_needs_wrap: bool,
 
-    /// Alternate grid.
-    alt_grid: Grid<Cell>,
+    /// Inactive grid.
+    inactive_grid: Grid<Cell>,
 
-    /// Alt is active.
-    alt: bool,
+    /// Inactive grid is active.
+    swapped: bool,
```

Comment:
This variable actually tracks if the alt screen is active. But I don't see why we would need it, we can just replace all usages of it with `self.mode.contains(TermMode::ALT_SCREEN)`.